classroom.rb

```diff
@@ -0,0 +1,12 @@
+class Classroom
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  attr_accessor :label
+
+  def add_student(student)
+    @students.push(student)
+  end
```

Comment:
- [ ] You have made a good attempt at setting up the classroom `has_many` students end of the relationship, however, you are yet to set the classroom for the student too. Kindly set the classroom of this student to `self`, like so:
 
  ```
  student.classroom = self
  ```